ucsb-cs156-w24/proj-happy-cows-w24-4pm-2
# Bharat UX Improvements to Leaderboard page

## Overview

In this PR, the back button on the leaderboard page was moved above the heading and aligned to the top right.

## Screenshots

![CleanShot 2024-03-07 at 11 19 07](https://github.com/ucsb-cs156-w24/proj-happy-cows-w24-4pm-2/assets/33289435/318654cf-69a6-4304-b853-ecabbb9e7c4e)

## Validation

1. Download this branch.
2. Run the project.
3. Navigate to the Leaderboard page.
4. Verify the location of the back button.

## Tests


## Linked Issues

Closes #8
